### PR TITLE
Update 05-RudolfZ11.md

### PR DESCRIPTION
**Pull request by RudolfZ11**
test

### CODE REVIEW

File: _slides/05-RudolfZ11.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+test
```
Comment (RudolfZ11) [inline _slides/05-RudolfZ11.md:6]: Text changed to "test"

**Review by RudolfZ11 — COMMENTED** [review]
Still testing

**Review by RudolfZ11 — COMMENTED** [review]
Trying approve

**Review by inessaw — APPROVED** [review]

**Comment by RudolfZ11** [comment]
Thank you

**Pull request merged**